core/domain/user_services.py

```diff
@@ -465,6 +464,53 @@ def validate(self):
                 'for a user.')
 
 
+class GaeIdToUserId(python_utils.OBJECT):
```

Comment:
The name suggests a mapping. Maybe call it GaeIdUserIdLink or similar, ditto for the model (similar to QuestionSkillLink + QuestionSkillLinkModel). Or alternatively: UserIdentifiers / UserIdentifiersModel (probably better since this is literally a container of user identifiers).

Reply:
Done.